packages/amplify-provider-awscloudformation/src/attach-backend.js

```diff
@@ -27,25 +27,18 @@ async function run(context) {
   }
   const { envName } = _.get(context, ['exeInfo', 'inputParams', 'amplify'], {});
   const { useProfile, configLevel } = _.get(context, ['exeInfo', 'inputParams', 'awscloudformation'], {});
-  if (!useProfile && (!configLevel || configLevel === 'amplifyAdmin')) {
-    if (appId && !envName && (await isAmplifyAdminApp(appId)).isAdminApp) {
-      throw new Error('Missing --envName <environment name> in parameters.');
-    } else if (appId && envName) {
-      // Check for existing Amplify Admin tokens
-      if (doAdminCredentialsExist(appId)) {
-        isAdminApp = true;
-      } else {
-        // Check if this is a Amplify Admin appId
-        const res = await isAmplifyAdminApp(appId);
-        isAdminApp = res.isAdminApp;
-        if (isAdminApp) {
-          // Admin app, go through login flow
-          try {
-            await adminLoginFlow(context, appId, envName, res.region);
-          } catch (e) {
-            context.print.error(`Failed to authenticate: ${e.message || 'Unknown error occurred.'}`);
-          }
-        }
+  if (!useProfile && (!configLevel || configLevel === 'amplifyAdmin') && appId) {
+    const res = await isAmplifyAdminApp(appId);
+    isAdminApp = res.isAdminApp;
+    if (isAdminApp) {
```

Comment:
What's the backup option here - if a customer is using cloud9/EC2 instances and have admin app enabled? Can they still use the AWS profile instead?

Reply:
It's not ideal, but you can workaround by changing the `configLevel` to `"project"` in the `<root>/amplify/.config/local-aws-info.json` file.

Reply:
A larger PR to address fallback scenarios is coming, but I didn't want to hold this fix up on it.

Reply:
Ah! Got it.